docker/zap-baseline.py

```diff
@@ -321,10 +321,23 @@ def main(argv):
         time.sleep(2)
 
         # Spider target
-        zap_spider(zap, target)
+        '''
+        Here the zap spider is supposed to only use a context if a context file is given
+        otherwise the inital spider funcionality without a context is used
+        '''
+        if context_file:
+            zap_spider(zap, target, os.path.splitext(os.path.basename(context_file))[0])
```

Comment:
The name of the context might not be the same as the name of the file, better get the name from ZAP (e.g. last one in the context list), it also avoids processing the name multiple times.

Reply:
Good catch.

Reply:
I'll use ZAP for that! Thx for the hint.

Reply:
@thc202 is there a special reason why to use the last context name from the context list?

Reply:
The last one is the one that was imported/added by the script.